lib/compat/wordpress-6.5/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -207,8 +196,7 @@ public function register_script_modules() {
 		 * @since 6.5.0
 		 */
 		public function add_hooks() {
-			add_action( 'wp_enqueue_scripts', array( $this, 'register_script_modules' ) );
-			add_action( 'wp_footer', array( $this, 'print_client_interactivity_data' ) );
+			add_filter( 'scriptmoduledata_@wordpress/interactivity', array( $this, 'print_client_interactivity_data' ) );
```

Comment:
Script enqueue should not have been deleted:
```suggestion
			add_action( 'wp_enqueue_scripts', array( $this, 'register_script_modules' ) );
			add_filter( 'scriptmoduledata_@wordpress/interactivity', array( $this, 'print_client_interactivity_data' ) );
```